### ISSUE TITLE
IP Lookup from notifications

### ISSUE THREAD

**Issue by ltGuillaume** [henrypp/simplewall#68]
It would be nice to be able to click on the external IP presented in a notification in order to lookup the IP with such a service.

Alternatively, a way to be able to copy the IP address to the clipboard would suffice.

**Comment by xEBFE** [comment]
I agree that just being able to copy an IP address from those notifications would be useful but I'm going to respectfully disagree with adding another feature to perform lookups. The reason being is that I feel simplewall's greatest strength is that it's extremely lightweight and simple (as the name implies).

**Comment by ltGuillaume** [comment]
I was just thinking of it opening up a privacy friendly website that looks up IP info. It's better than people entering the IP's in Google every time, not very privacy friendly.

**Comment by pwn0r** [comment]
not trying to advertise or anything but for lookups I would recommend ipinfo.io
